app/2.0/docs/devguide/gesture-events.md

```diff
@@ -188,12 +188,52 @@ to be garbage collected.
 
 ```
 
-## Gestures and scroll direction
+## Native browser gesture handling {#gestures-and-scroll-direction}
+
```

Comment:
@azakus this section is new, replacing some stale 1.x stuff.

Reply:
LGTM